spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -401,6 +401,7 @@ def start_client(self, language):
                 logger.info("Starting LSP client for {}...".format(language))
                 language_client['instance'].start()
                 language_client['status'] = self.RUNNING
+                started = True
```

Comment:
@ccordoba12 is this really needed? If so, maybe adding a comment could be worthy

Reply:
Good eye! It's not directly related to this PR but I found it while testing completion and linting for languages other than Python.

I updated the PR description to include it and will add a couple of comments to clarify why it's necessary.